concierge/service/src/main/java/org/eclipse/ditto/concierge/service/enforcement/DefaultCreationRestrictionEnforcer.java

```diff
@@ -0,0 +1,129 @@
+/*
+ * Copyright (c) 2021 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.eclipse.ditto.concierge.service.enforcement;
+
+import org.eclipse.ditto.concierge.service.common.CreationRestrictionConfig;
+import org.eclipse.ditto.concierge.service.common.EntityCreationConfig;
+import org.eclipse.ditto.internal.utils.akka.logging.DittoLoggerFactory;
+import org.slf4j.Logger;
+
+import javax.annotation.concurrent.Immutable;
+import java.util.List;
+import java.util.Set;
+import java.util.regex.Pattern;
+
+/**
+ * Default implementation for {@link CreationRestrictionEnforcer} based on the configuration from {@link EntityCreationConfig}.
+ */
+@Immutable
+public class DefaultCreationRestrictionEnforcer implements CreationRestrictionEnforcer {
+
+    private static final Logger log = DittoLoggerFactory.getThreadSafeLogger(DefaultCreationRestrictionEnforcer.class);
```

Comment:
```suggestion
    private static final ThreadSafeDittoLogger LOG = DittoLoggerFactory.getThreadSafeLogger(DefaultCreationRestrictionEnforcer.class);
```

Please use `ThreadSafeDittoLogger LOG` instead - that way we can directly invoke
`LOG.withCorrelationId(context.getDittoHeaders()).debug("....")`